DMTN-226.tex

```diff
@@ -202,16 +231,57 @@ \section{Implementing the Rubin AFS with ANTARES} \label{sec:implementation}
 A follow-on video conference was held in April 2022.
 Participants in the latter teleconference agreed that the simplest and thus most cost-effective course of action would be to explore how the ANTARES-hosted filtering service could provide the functionality of the Rubin AFS.
 
-We identify the following open questions which should be resolved to finalize this understanding:
+Subsequent dialogue achieved convergence on the following topics.
+
+\subsection{Alert Throughput}
+
+ANTARES report cloud scaling tests run in GCP show that the ANTARES system can handle the average LSST alert rate (10,000 alerts every 39 seconds) with 100--150 nodes.
+Performance optimization is ongoing.
+
+\subsection{Access to Rubin Data Rights Holders}
+
+Currently ANTARES services are open to all users.  
+The ANTARES team expect to easily support the number of Rubin Data Rights holders (20) specified in DMS-REQ-0343.
```

Comment:
I think the number of alerts per user is 20 and the number of simultaneous users is 100. We should hyperlink the requiremets to the LSE-61.

Reply:
Good catch, Bob, thanks.